### ISSUE TITLE
TO-DO list

### ISSUE THREAD

**Issue by jetnisa** [Rachelouve/Intro_R_HWM1#1]
For the 15/03 : 
Jetnisa : do the first two sections of the homework and individual part of section 3. 
mehdi : Add a video to the introduction, do the individual part of point 3. Part 4 : start by doing the first 3 points of this section. 
Rachel and Neila : do the individual part for the third section.
For the 17/03: we should all have finished our parts and finish the fourth section.
18/03 : Do Color box and references all together so we finish the homework.

**Comment by Hirari98** [comment]
yes finished my part just need to update the click button and the color box

**Comment by Neila03** [comment]
I finished my part and we will finish to do references now.
